camp/leo-miniProject/src/main/java/frc/robot/commands/XBoxLeftBumper.java

```diff
@@ -0,0 +1,51 @@
+// ************************************************************
+// Bishop Blanchet Robotics
+// Home of the Cybears
+// FRC - Reefscape - 2025
+// File: XboxLeftBumper.java
+// Intent: Xbox left bumpers
+// ************************************************************
+package frc.robot.commands;
+import frc.robot.Constants;
+import frc.robot.subsystems.TalonMotorSubsystem;
+import edu.wpi.first.wpilibj2.command.Command;
+
+/** An example command that uses an example subsystem. */
+public class XBoxLeftBumper extends Command {
+  private final TalonMotorSubsystem motorSubsystem;
+
+
+
+  //creates a new xbox left bumper command
+   
+  public XBoxLeftBumper(TalonMotorSubsystem subsystem) {
+    this.motorSubsystem = subsystem;
+    // Use addRequirements() here to declare subsystem dependencies.
+    addRequirements(subsystem);
+  }
+
+  // Called when the command is initially scheduled.
```

Comment:
If you don't have anything to put in initialize(), then you don't need to include it.